libevmasm/AssemblyItem.cpp

```diff
@@ -90,6 +90,10 @@ std::pair<std::string, std::string> AssemblyItem::nameAndData(langutil::EVMVersi
 	case JumpF:
 	case RetF:
 		return {instructionInfo(instruction(), _evmVersion).name, ""};
+	case SwapN:
+		return {instructionInfo(instruction(), _evmVersion).name, util::toString(static_cast<size_t>(data())) };
+	case DupN:
+		return {instructionInfo(instruction(), _evmVersion).name, util::toString(static_cast<size_t>(data())) };
```

Comment:
The cast isn't even necessary here, is it?
```suggestion
	case SwapN:
	case DupN:
		return {instructionInfo(instruction(), _evmVersion).name, toString(data())};
```